website/app/templates/head.hbs

```diff
@@ -45,6 +45,7 @@ We've made some small customizations to simplify this compared to the default te
 {{! BEGIN TWITTER }}
 
 <meta name="twitter:card" content="summary" />
```

Comment:
```suggestion
<meta name="twitter:card" content="summary_large_image" />
```

From my experience, this works more accurately.

Reply:
Done